README.md

```diff
@@ -1,20 +1,88 @@
 # Flow Data Provisioning Service
 
+The Flow Data Provisioning Service aims at providing a scalable and efficient way to access the execution state of past
+and live sporks, as well as implementing the Rosetta API standard.
+
+Past sporks are loaded from the filesystem using Flow's Write-Ahead Log to replay the changes to all registries, and
+updates are pushed directly to the DPS by the execution nodes in order to handle live networks.
```

Comment:
`For the live spork index, the DPS subscribes to an execution node and receives state trie and transaction event updates over the network.`